lib/utils/cli.go

```diff
@@ -295,6 +295,16 @@ func InitCLIParser(appName, appHelp string) (app *kingpin.Application) {
 	return app.UsageTemplate(createUsageTemplate())
 }
 
+// InitHiddenCLIParser initializes a `kingpin.Application` that does not terminate the application
+// or write any usage information to os.Stdout.
+func InitHiddenCLIParser() (app *kingpin.Application) {
```

Comment:
This godoc explains "what", but not "why".  Can you add a rationale and some information how it relates to  `utils.InitCLIParser`?